docs/includes/manifest-guidance.md

```diff
@@ -1,7 +1,12 @@
 > [!TIP]
 > If you'll be testing your add-in across multiple environments (for example, in development, staging, demo, etc.), we recommend that you maintain a different manifest file for each environment. In each manifest file, you can:
+> 
 > - Specify the URLs that correspond to the environment.
-> - Customize metadata values like `DisplayName` and labels within `Resources` to indicate the environment, so that end users will be able to identify a sideloaded add-in's corresponding environment. 
+> - Customize metadata values so that end users will be able to identify a sideloaded add-in's corresponding environment. For example:
```

Comment:
```suggestion
> - Customize metadata values so that end users are able to identify a sideloaded add-in's corresponding environment. For example:
```